src/clients/secretsManagerService.js

```
@@ -0,0 +1,92 @@
+} from '@aws-sdk/client-secrets-manager';
+import logger from '../utils/logger.js';
+
+export const createAwsSecretsManagerClient = () => {
+  logger.debug('Creating AWS SecretsManagerClient');
+  return new SecretsManagerClient({ region: 'us-east-1' });
+};
+
+export class SecretsManagerService {
+  constructor(secretsManagerClient) {
+    logger.debug('SecretsManagerService constructor called');
+    if (!secretsManagerClient) throw new Error('SecretsManagerClient is required');
```

ESLint

[error] Expected { after 'if' condition.

(curly)